src/pages/index.tsx

```diff
@@ -78,8 +78,8 @@ const Page = () => {
         socialLinks={SOCIAL_LINKS as NavMenuItem[]}
         currentSite={'Docs'}
       />
-      <UniversalNav blend={true} />
-      <Hero>
+      <SecondaryNav />
+      <Hero className="margin-top-lg">
```

Comment:
Hi @oscarcao, I'm not sure if this is the best way to do it as now we're adding extra margin to this component on all screen sizes. I think it would be better if we add a media query to the `Hero` styled component instead. The width when things start to change is `975px`. Is this something you are okay with changing or did you want me to do it?

Reply:
@oscarcao if you could fix this up, we can accept it!

Reply:
Hi @timngyn and @rachelnabors

Have you guys had a look at how it looks on desktop with the added spacing at the top? I think you guys would prefer it with the extra spacing.

Cheers
Oscar

Reply:
Hi @oscarcao, thanks for the response, we did take a look and really appreciate your work on this! 

The issue I was trying to point out is that the spacing is inconsistent when the navbar switches to a mobile view and the desktop view. For instance, here in the video, I'm adjusting the width down and we can see that the "Amplify Documentation" title is jumping up and down even though the height of the navbar and the secondary nav are still the same.

https://user-images.githubusercontent.com/54393192/199626608-a49b2393-5341-4be7-acdd-e176f2e8b532.mov

I am okay with the extra spacing on desktop, but in my opinion it should be consistent between the different screen sizes. If you want to keep this extra margin then I think we can still add a media query to include this extra spacing like I mentioned earlier

Reply:
No worries. Have updated implementation so the hero section does not jump around when break points change.